spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'root page features' do
+  include Capybara::DSL
+  let!(:user) do
+    new_user = User.new(
+      id: 1,
+      name: 'John Doe',
+      email: 'john.doe+2@example.com',
+      bio: "Hello, I'm John!",
+      posts_counter: 0,
+      photo: Rack::Test::UploadedFile.new(
+        Rails.root.join('app', 'assets', 'images', 'profile.jpg'),
+        'image/jpeg'
+      )
+    )
+    new_user.save
+    new_user
+  end
+
+  it 'displays the user name and profile image' do
+    visit '/user'
+
+    expect(page).to have_content(user.name)
+  end
+
+  it 'displays the first 3 posts' do
+    visit '/user'
+
```

Comment:
Good job having the set up for this user show page, however there are a couple of tests yet to be written such as:

- Tests that assert you can see the user's profile picture.
- Tests that asserts you can see the number of posts the user has written.
- Tests that asserts you can see the user's bio.
- Tests that asserts you can see a button that lets me view all of a user's posts.
- Tests that redirects to the post's show page when you click on a user's post.
- Tests that redirects to the user's post's index page when you click on a see all posts

Consider writing the above mentioned test in order meet up with the milestone requirements